src/components/RocketsList.jsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import RocketItem from 'components/RocketItems';
+
+const RocketsList = () => {
+  const { rocketItems, isLoading, error } = useSelector((state) => state.rockets);
```

Comment:
Excellent use of useSelector to call the state of your rocketsSlice, however, I would suggest to use `(state) => state.rockets` within the slice so you do not repeat yourself everytime you need to call it

Reply:
I'm not sure, but I guess I don't understand the above-required change. Was it a requirement in the project? Please I would like to have more explicit details. 🙏